### PR TITLE
Create Cancel Dialog for Edit and Create Dialogs

### PR DESCRIPTION
**Pull request by VitalyiKhlivnyi**
Create Cancel Dialog for Edit and Create Dialogs
Edit README.md

## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/browse/LEAR-51)

## Code reviewers

- [ ] @ashatna
- [ ] @ChristinaPastushchak
- [ ] @dotiurin
- [ ] @iryn34
- [ ] @tarassito
- [ ] @VasilHamulyak
- [ ] @VitalyiKhlivnyi
- [ ] @YurijH

### Second Level Review

- [ ] @ushevtsiv
- [ ] @lhalam
- [ ] @ssOleg

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: eventually/static/src/containers/curriculum/CurriculumDialog.js [hunk 1]
```diff
@@ -87,6 +93,23 @@ export default class CurriculumDialog extends React.Component {
...
+    handleRequestClose = () => {
+        if ((this.state.title != '') ||
+            (this.state.description != '')) {
+            this.setState({openCancelDialog: true});
+        }
+        else this.handleClose();
+    };
+
     render() {
         const disable = !(this.state.titleIsValid&this.state.descriptionIsValid);
```
Comment (iryn34) [inline eventually/static/src/containers/curriculum/CurriculumDialog.js]: use && instead of &

**Review by iryn34 — DISMISSED** [review]

**Review by iryn34 — APPROVED** [review]

**Review by lhalam — APPROVED** [review]

File: eventually/static/src/containers/topicItemList/EditItemDialog.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ import SvgIcon from 'material-ui/SvgIcon';
 import ModeEdit from 'material-ui/svg-icons/editor/mode-edit';
 import SelectField from 'material-ui/SelectField';
 import MenuItem from 'material-ui/MenuItem';
-import { putEditItemService } from 'src/containers';
+import { putEditItemService, CancelDialog } from 'src/containers';
```
Comment (tarassito) [inline eventually/static/src/containers/topicItemList/EditItemDialog.js:10]: space after { ?

File: eventually/static/src/containers/topicList/EditTopicDialog.js [hunk 3]
```diff
@@ -7,7 +7,7 @@ import SvgIcon from 'material-ui/SvgIcon';
 import ModeEdit from 'material-ui/svg-icons/editor/mode-edit';
 import SelectField from 'material-ui/SelectField';
 import MenuItem from 'material-ui/MenuItem';
-import { putEditTopicService } from 'src/containers';
+import { putEditTopicService, CancelDialog } from 'src/containers';
```
Comment (tarassito) [inline eventually/static/src/containers/topicList/EditTopicDialog.js:10]: you know what I mean )

File: eventually/static/src/containers/voting/CreateCustomVote.js [hunk 4]
```diff
@@ -4,8 +4,8 @@ import FlatButton from 'material-ui/FlatButton';
 import TextField from 'material-ui/TextField';
 import FloatingActionButton from 'material-ui/FloatingActionButton';
 import ContentAdd from 'material-ui/svg-icons/content/add';
-import { postVote } from './VoteService';
-import { postAnswer } from './VoteService';
+import { postVote, postAnswer } from './VoteService';
+import { CancelDialog } from 'src/containers';
```
Comment (tarassito) [inline eventually/static/src/containers/voting/CreateCustomVote.js:8]: ?

**Pull request merged**